### PR TITLE
Import SwiftFormat

### PR DESCRIPTION
**Pull request by 417-72KI**
There are some discussions about formatting (e.g. https://github.com/nerdishbynature/octokit.swift/pull/116#discussion_r531528690)

I add [SwiftFormat](https://github.com/nicklockwood/SwiftFormat) via [Mint](https://github.com/yonaskolb/Mint), and draft configuration.

### CODE REVIEW

File: .swiftformat [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# file options
+
+--exclude Tests/LinuxMain.swift,Package.swift
+
+# format options
+--swiftversion 4.2
```
Comment (MaxDesiatov) [inline .swiftformat:6]: Does 4.0 work here? Our `Package.swift` specifies 4.0

**Review by MaxDesiatov — APPROVED** [review]
It's a very welcome change 👍

The only comment I have is about the Swift version, which is inconsistent with what's specified in `Package.swift`.

File: .swiftformat (hunk 1 above)
Comment (417-72KI) [reply .swiftformat]: I'm not sure but I think it will work.

I tried adding matrix to run with Swift 4.0 on Travis🙂

Comment (417-72KI) [reply .swiftformat]: CI failed on 4.0 and 4.1. 
It seems to be required 4.2 on [RequestKit](https://github.com/nerdishbynature/RequestKit/blob/master/Package.swift#L1)🤔

File: Package.swift [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.0
+// swift-tools-version:4.2
```
Comment (417-72KI) [inline Package.swift:1]: Match to [RequestKit](https://github.com/nerdishbynature/RequestKit/blob/master/Package.swift#L1)

**Comment by pietbrauer** [comment]
Thanks for the config file, @417-72KI. This was too hard to rebase so I made a new branch with GitHub Actions in #130. And sorry this took so long.

**Pull request closed without merging**